core/modules/filter/filter.module

```diff
@@ -2542,3 +2545,14 @@ function _filter_image_align($text) {
 /**
  * @} End of "Standard filters".
  */
+
+/**
+ * Implements hook_autoload_info().
+ */
+function filter_autoload_info() {
+  return array(
+    // Views handlers.
+    'views_handler_field_file_uri' => 'views/views_handler_field_file_uri.inc',
```

Comment:
This should be in File module, not Filter module.

Reply:
I have now removed this from the Filter module and modified the views handler in the File module. As this is a modification that is only relevant to image files, I am wondering whether I should make bigger changes, but I am very much a beginner with views handlers.